### DISCUSSION TITLE
no overspeed warning

### DISCUSSION THREAD

**Discussion by OldGroof** [magknight/787aviatorsTracker#519] [Ideas]
**Describe the bug**
There's no overspeed warning when the plane overspeeds

**To Reproduce**
Fly above Vne

**Expected behavior**
There should be a warning. Not sure what it sounds like on the 787 but there should be one.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Windows 10
 - X-Plane 11.50b17
 - Version 1.6.0cygnus9

**Comment by aeroniemi** [comment]
this was an intentional descision because the max speed isn't entirely accurate - but we could probably change that

**Reply by Simleybob** [reply]
Please change it!
